lib/hanami/events/formatter/json.rb

```diff
@@ -0,0 +1,17 @@
+require 'json'
+
+module Hanami
+  module Events
+    class Formatter
+      class Json
+        def initialize(data)
+          @data = data
```

Comment:
Yep, looks good! `events_meta` makes sense. 
And for `meta` method implementation we could rename `title` to `event_name`. To have consistent naming.